tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -147,12 +147,20 @@ internal static partial class ConfigurationKeys
         public const string LogsInjectionEnabled = "DD_LOGS_INJECTION";
 
         /// <summary>
+        /// This parameter is obsolete and should be replaced by `DD_TRACE_RATE_LIMIT`.
         /// Configuration key for setting the number of traces allowed
         /// to be submitted per second.
         /// </summary>
         /// <seealso cref="TracerSettings.MaxTracesSubmittedPerSecond"/>
         public const string MaxTracesSubmittedPerSecond = "DD_MAX_TRACES_PER_SECOND";
```

Comment:
Shall we mark this as `[Obsolete]`?

Reply:
I tried and got a build error as I still use it. Didn't think much more about it as it's internal.

Reply:
Yeah, you should either explicitly use the new one, or add a `pragma` if we still want to test using it. IMO, we should be explicit with the `[Obsolete]` where possible, but not too bothered

Reply:
Makes sense. Fixed in 1c1a1f2e0ff3724591fa55225fef5d077f24a0b7